main.go

```diff
@@ -34,6 +37,11 @@ const (
 	DEFAULT_KEY = "default"
 )
 
+const (
+	resolvConfLocation = "/etc/resolv.conf"
+	ipLocalhost        = `^nameserver\s+((127\.([0-9]{1,3}\.){2}[0-9]{1,3})|(::1)$)`
```

Comment:
You can do `^nameserver\s+((127(\.[0-9]{1,3}){3})|(::1)$)` instead.

Reply:
@yasker Hi Yasker, 127.0.0.1, the last number don't have the dot.

Reply:
You read it wrong.

You grouped it as ,`127.[XXX.]{repeat twice}XXX`

What you should do it as `127[.XXX]{repeat three times}`, as in my regex.

Reply:
Just updated to use ^nameserver\s+((127(\.[0-9]{1,3}){3})|(::1)$)